changelogs/unreleased/5211-cleverhu

```diff
@@ -1 +0,0 @@
-fix run preHook and postHook on completed pods
```

Comment:
Is this changelog meant to be deleted completely rather than a single line?

Reply:
The file is deleted in the commit.

Reply:
noted.